src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -1131,10 +1131,12 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         }
 
         and: "The new and old dst switches pass switch validation"
-        [dstSwitch, newDstSwitch]*.dpId.each { switchId ->
-            with(northbound.validateSwitch(switchId)) { validation ->
-                validation.verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
-                validation.verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
+        Wrappers.wait(RULES_DELETION_TIME / 2) {
```

Comment:
why?

Reply:
it is difficult to reproduce. Probably we need to run it around 30 times and one them is failed.
it is a rare case, but sometimes it happens.

Reply:
I often see failed build on jenkins due to this test

Reply:
Ok, we can try, but the excess rules you're seeing may be due to #3256, and this test is just the first in the sequence that does the rules verificaiton and shows the problem

Reply:
this issue is not caused by other test.
I've rerun this test locally a lot of times and the issue sometimes is reproduced